mava/networks.py

```diff
@@ -107,13 +107,13 @@ class FeedForwardActor(nn.Module):
     @nn.compact
     def __call__(self, observation: Observation) -> distrax.DistributionLike:
         """Forward pass."""
-        x = observation.agents_view
 
-        x = self.torso(x)
+        obs_embedding = self.torso(observation.agents_view)
 
-        x = self.action_head(x, observation.action_mask)
-
-        return x
+        # We pass both the observation embedding and the observation object to the action head
+        # since the observation object contains the action mask and other potentially useful
+        # information.
```

Comment:
Nit: can you add this to the docs of the `__call__` of `DiscreteActionHead`

Reply:
Done :call_me_hand: